src/Processor/CheckoutProcessor.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ *NOTICE OF LICENSE
+ *
+ *This source file is subject to the Open Software License (OSL 3.0)
+ *that is bundled with this package in the file LICENSE.txt.
+ *It is also available through the world-wide-web at this URL:
+ *http://opensource.org/licenses/osl-3.0.php
+ *If you did not receive a copy of the license and are unable to
+ *obtain it through the world-wide-web, please send an email
+ *to license@prestashop.com so we can send you a copy immediately.
+ *
+ *DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ *versions in the future. If you wish to customize PrestaShop for your
+ *needs please refer to http://www.prestashop.com for more information.
+ *
+ *@author INVERTUS UAB www.invertus.eu  <support@invertus.eu>
+ *@copyright SIX Payment Services
+ *@license   SIX Payment Services
+ */
+
+namespace Invertus\SaferPay\Processor;
+
+use Cart;
+use Invertus\SaferPay\Config\SaferPayConfig;
+use Invertus\SaferPay\EntityBuilder\SaferPayOrderBuilder;
+use Invertus\SaferPay\Factory\ModuleFactory;
+use Invertus\SaferPay\Repository\SaferPayOrderRepository;
+use Invertus\SaferPay\Service\SaferPayInitialize;
+use Order;
+use SaferPayOrder;
+
+class CheckoutProcessor
```

Comment:
https://github.com/Invertus/klarnapayment/blob/e109b428b4aef42e8fd9ad5654faba6d0a7e97f9/src/Core/Payment/Processor/CheckoutProcessor.php#L47 try it out like this, as currently if you fail to call one of these functions - everything will fail.